run_bioblend_tests.sh

```diff
@@ -146,6 +146,9 @@ else
 fi
 export BIOBLEND_GALAXY_URL=http://localhost:${GALAXY_PORT}
 
+# Set variable for ToolShed tests
+export BIOBLEND_TOOLSHED_URL=https://testtoolshed.g2.bx.psu.edu/
+
```

Comment:
The CI now uses this script, so if you set `BIOBLEND_TOOLSHED_URL` here, all CI jobs will run the TS tests.

```suggestion
```